ruddock/modules/rotation/routes.py

```diff
@@ -0,0 +1,43 @@
+import json
+import flask
+import io
+
+from ruddock.resources import Permissions
+from ruddock.decorators import login_required
+from ruddock.modules.rotation import blueprint, helpers
+
+
+@blueprint.route('/')
+@login_required(Permissions.ROTATION)
+def show_portal():
+  dinner_list = helpers.get_dinners()
+  return flask.render_template('portal.html',
+    dinner_list=dinner_list)
+
+@blueprint.route('/dinner/<dinner_id>')
+@login_required(Permissions.ROTATION)
+def show_dinner_list(dinner_id):
+  prefrosh_list = helpers.get_prefrosh_by_dinner(dinner_id)
+  return flask.render_template('directory.html',
+    prefrosh_list=prefrosh_list, dinner_id=dinner_id)
+
+@blueprint.route('/images/<prefrosh_id>')
+@login_required(Permissions.ROTATION)
+def serve_image(prefrosh_id):
+  img_contents = helpers.get_image_contents(prefrosh_id)
+  return flask.Response(io.BytesIO(img_contents), '200 OK', None, 'image/jpeg', 'image/jpeg')
```

Comment:
kwargs should be passed in by the `kwarg=value` syntax, to be more readable (it's position isn't helpful unless you look at the function's signature).